src/Administration/Resources/app/administration/src/module/sw-media/component/sw-media-library/index.js

```diff
@@ -238,7 +238,6 @@ Component.register('sw-media-library', {
                 .addAssociation('tags')
                 .addAssociation('productMedia.product')
                 .addAssociation('categories')
-                .addAssociation('productManufacturers.products')
```

Comment:
i think the `productManufacturers` association is still needed, but we should be able to remove the `products` association of the `manufacturers`, i think that should already improve the performance greatly in your case.

Reply:
yes, this is exactly what I did :-) the manufacturers are still necessary, but there are only few of them - their products are the problem.

Reply:
but you still need `.addAssociation('productManufacturers')` otherwise it won't load the manufacturer association.